ui/lib/pki/addon/routes/tidy/auto.js

```diff
@@ -1,3 +1,8 @@
 import Route from '@ember/routing/route';
 
-export default class PkiTidyAutoRoute extends Route {}
+export default class PkiTidyAutoRoute extends Route {
```

Comment:
This serves as the base route for tidy/auto/config as well which would render the template on the config template, so I created `tidy/auto/index` to handle what we actually want to see on this route and the config and view routes can share this model.